product_available_by_warehouse/models/product.py

```diff
@@ -40,3 +40,11 @@ def get_product_available_by_warehouse(self):
                 'outgoing_qty': outgoing_qty,
             }
         return res
+
+    @api.multi
+    def _get_domain_locations(self):
```

Comment:
Hello @suniagajose 

Field ```qty_available``` shows quantities of location that are not internal locations, how it can see [here](https://www.youtube.com/watch?v=6RxHck7KGtw&feature=youtu.be)